testcontainers/localstack.py

```diff
@@ -24,7 +24,7 @@ class LocalStackContainer(DockerContainer):
         localstack = LocalStackContainer(image="localstack/localstack:0.11.3")
```

Comment:
This is an older version. Update to latest? Remove image kwarg and use default?

```suggestion
        localstack = LocalStackContainer(image="localstack/localstack:0.11.4")
```